sql/parser/builtins.go

```diff
@@ -943,6 +961,89 @@ func datumToString(datum Datum) (string, error) {
 	return "", fmt.Errorf("argument type unsupported: %s", datum.Type())
 }
 
+func regexpReplace(s, pattern, to, flags string) (Datum, error) {
```

Comment:
it'd be good to call this `sqlFlags` to clarify that it refers to sql-syntax flags